pkgs/tools/security/dnsrecon/default.nix

```diff
@@ -41,3 +47,4 @@ python3.pkgs.buildPythonApplication rec {
     maintainers = with maintainers; [ c0bw3b fab ];
   };
 }
+
```

Comment:
```suggestion
```
or is there a reason you added this?

Reply:
No, reason. Could come from the editor or I forgot to run `nixpkgs-fmt`.

Reply:
can you remove it again? It should be good to go after that.